packages/vue/src/components/authenticator.vue

```diff
@@ -77,17 +64,13 @@ const emit = defineEmits([
   'verifyUserSubmit',
   'confirmVerifyUserSubmit',
 ]);
-const machine = createAuthenticatorMachine();
 
-const service = useInterpret(machine);
-let unsubscribeHub: () => void;
 let unsubscribeMachine: () => void;
 
-const { state, send } = useActor(service);
-useAuth(service);
-
```

Comment:
Previously, this file used `useActor`, `useAuth`, and `useAuthenticator` in one file. This actually had a race condition based on which one finishes updating the facade value.

For example, `state.matches('signUp')` might execute faster than `useAuthenticator` finishes computing the facade value. This could lead to UI rendering early before contexts are ready.